headers/data/arm9.h

```diff
@@ -182,6 +182,7 @@ extern uint8_t GAME_MODE;
 extern struct global_progress* GLOBAL_PROGRESS_PTR;
 extern struct adventure_log* ADVENTURE_LOG_PTR;
 extern void* ITEM_TABLES_PTRS_1[26];
+extern void* MONSTER_DATA_TABLE_PTR;
```

Comment:
Let's make this `undefined*` if it's a fixed width struct we don't know the layout of

Reply:
Fixed.